policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/expense_rate.yaml

```diff
@@ -0,0 +1,27 @@
+description: The Department for Education converts childcare hours into expenses for extended childcare entitlement at these rates, depending on the child's age.
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-GBP
+  label: Extended childcare entitlement hourly rates by age
+  reference:
+    - title: Different Rates for Extended Childcare Entitlement
+      href: 
+        - https://www.nurseryworld.co.uk/content/news/dfe-confirms-extended-30-hours-funding-rates-but-sector-says-they-dont-reflect-the-real-delivery-cost/
```

Comment:
There's nowhere in the codebase that we make `href` a dictionary or list. It's always a string with the URL